fides/templates/fides/fides-config.yaml

```diff
@@ -12,9 +12,7 @@ data:
     db_index = 0
     
     [security]
-    {{- if .Values.ingress.enabled }}
-    cors_origins = [{{ printf "https://%s" .Values.ingress.hosts.privacyCenter | quote }}, {{ printf "https://%s" .Values.ingress.hosts.fides | quote }}]
-    {{- end }}
+    cors_origins = {{ include "fides.corsOrigins" . | trim }}
```

Comment:
This should be fine, but are you sure you don't want to just do this using ENV variables in general?

I just feel like those are "easier" to see and troubleshoot in almost every circumstance for devops things.

Reply:
This is one that I'll revisit. I'd rather not have _any_ `toml` file, but since this is where it was already configured, going to do it in a future enhancement.